kqueen/engines/gce.py

```diff
@@ -189,6 +204,43 @@ def resize(self, node_count, **kwargs):
 
         return True, None
 
+    def set_network_policy(self, network_provider='CALICO', enabled=False):
```

Comment:
please, add a docstring with parameter and return values description and why it is disabled by default